src/subsites/SubsiteMemberReportExtension.php

```diff
@@ -1,8 +1,13 @@
 <?php
+namespace SilverStripe\SecurityReport;
```

Comment:
perhaps add a `return;` at the top here to stop the class from being loaded at all?
Better than loading this unsupported class, and cleaner than commenting out the whole file